src/vs/workbench/parts/preferences/browser/settingsTree.ts

```diff
@@ -1062,14 +1061,11 @@ export class SettingsRenderer implements ITreeRenderer {
 			template.descriptionElement.innerText = element.description;
 		}
 
+		// List row label overrides labeling - is not full label must use local label
+		template.containerElement.parentElement.removeAttribute('aria-label');
```

Comment:
I don't understand the comment. Is this removing the aria label from SettingsAccessibilityProvider?

Reply:
It removes the label from the tree  row.  This label interferes with the local label the input element.
With voiceover it causes a duplicate label output with checkbox elements.

I am going to work on the general issue of role types and labeling variations for both tree and list.
Once I get that in we could do more generalization for setting and controlling these attributes as opposed to hack overrides.

Reply:
I should do this within SettingsAccessibilityProvider if checkbox.